src/openforms/forms/tests/test_utils.py

```diff
@@ -262,7 +262,7 @@ def test_convert_logic_with_selectboxes(self):
         rule = form_step.form.formlogic_set.first()
 
         self.assertEqual(
-            {"==": [{"var": "test1.a"}, True]},
+            {"==": [{"var": "test1.a"}, "a"]},
```

Comment:
this is wrong. selectboxes have a data value format of:

```json
{
   "a": true,
   "b": false
}
```

and that must be reflected correctly in the json logic trigger